src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -1929,6 +1932,19 @@ JRT_ENTRY(void, Deoptimization::uncommon_trap_inner(JavaThread* current, jint tr
 
     JFR_ONLY(post_deoptimization_event(nm, trap_method(), trap_bci, trap_bc, reason, action);)
 
+    if(Log(deoptimization)::is_info()) {
+      LogMessage(deoptimization) lm;
+      auto tm = trap_method();
+      lm.info("Deoptimization start {");
```

Comment:
Do we use this brace style for other log messages? I'm not really seeing the need for a start/end delimiter here.